components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -794,6 +794,18 @@ function _Set-Environment {
 function Invoke-PrepareWrapper {
     Write-BuildLine "Preparing to build"
     Push-Location "$HAB_CACHE_SRC_PATH\$pkg_dirname"
+    foreach($dir in $pkg_lib_dirs) {
+        New-Item $dir -ItemType Directory -Force | Out-Null
```

Comment:
add `-ErrorAction SilentlyContinue` to make sure it does not blow up if the dir already exists